app/presenters/html_publication_presenter.rb

```diff
@@ -13,6 +13,18 @@ def contents
     content_item["details"]["headings"].html_safe
   end
 
+  def isbn
+    content_item["details"]["isbn"] if content_item["details"]["isbn"].present?
```

Comment:
I'm not sure `if content_item["details"]["isbn"].present?` is necessary, as `content_item["details"]["isbn"]` will return nil if not present.